bin/build-windows.rb

```diff
@@ -137,6 +140,16 @@ def clone_and_rynsc_private_repo(fork, ref, hostname, ssh_key)
 Kernel.system("set -vx;#{ssh_command} \"source .bash_profile ; mkdir -p #{archive_dest}/#{pxp_zipname}/modules ; cp -r /home/Administrator/cpp-pcp-client/release/bin #{archive_dest}/#{pxp_zipname} ; cp -r /home/Administrator/pxp-agent/release/bin #{archive_dest}/#{pxp_zipname}; cp -r /home/Administrator/pxp-agent/modules/pxp-module-puppet* #{archive_dest}/#{pxp_zipname}/modules \"")
 fail "Copying source files for packaging failed" unless $?.success?
 
+if !(PRESERVE_TESTFILES)
+  # Remove test files prior to zipping
+  # facter
+  Kernel.system("set -vx;#{ssh_command} \"source .bash_profile ; rm #{archive_dest}/#{facter_zipname}/bin/*_test.exe ; rm #{archive_dest}/#{facter_zipname}/bin/lth_cat.exe ;\"")
+  fail "Cleaning source files prior to packaging failed" unless $?.success?
+  # pxp-agent
+  Kernel.system("set -vx;#{ssh_command} \"source .bash_profile ; rm #{archive_dest}/#{pxp_zipname}/bin/*-unittests.* ; rm #{archive_dest}/#{pxp_zipname}/bin/lth_cat.exe ;\"")
+  fail "Cleaning source files prior to packaging failed" unless $?.success?
+end
+
```

Comment:
I'm not sure if this will cause problems, but we are planning on moving leatherman to a distinct puppet-agent component. Will test executables be reintroduced once that happens? /cc @MikaelSmith